### ISSUE TITLE
missing folders files of testsuite in tarball in release of -1.4

### ISSUE THREAD

**Issue by idella** [django-nose/django-nose#227]
running with 
$ python testapp/runtests.py

appears to work well enough.

Running the ./runtests.sh yields

```bash
PASS: normal settings
PASS: normal settings (--help)
PASS: django_nose.run_tests format
PASS: django_nose.run_tests format (--help)
PASS: via run_tests API
PASS: via run_tests API (--help)
PASS: with plugins
PASS: with plugins (--help)
FAIL: unittests
E
======================================================================
ERROR: Failure: ImportError (No module named 'unittests')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python3.4/site-packages/nose/failure.py", line 39, in runTest
    raise self.exc_val.with_traceback(self.tb)
  File "/usr/lib64/python3.4/site-packages/nose/loader.py", line 409, in loadTestsFromName
    module = resolve_name(addr.module)
  File "/usr/lib64/python3.4/site-packages/nose/util.py", line 312, in resolve_name
    module = __import__('.'.join(parts_copy))
nose.proxy.ImportError: No module named 'unittests'
-------------------- >> begin captured logging << --------------------
django.db.backends.schema: DEBUG: CREATE TABLE "django_migrations" ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT, "app" varchar(255) NOT NULL, "name" varchar(255) NOT NULL, "applied" datetime NOT NULL); (params [])
--------------------- >> end captured logging << ---------------------

----------------------------------------------------------------------
Ran 1 test in 0.001s

FAILED (errors=1)
```

The folder unittests in the repo simply didn't make it to the tarball. I don't know whether this is by accident or intent, but the running by ./runtests.sh isn't equipped to pass.

**Comment by jwhitlock** [comment]
This is fixed in master, 112f48f, and will be included in the 1.4.2 release.

**Comment by idella** [comment]
thanks
